ThirdPartyAdapters/unity/unity/src/main/java/com/google/ads/mediation/unity/UnityRewardedAd.java

```diff
@@ -151,6 +164,9 @@ public void showAd(@NonNull Context context) {
       Log.w(TAG, "Unity Ads received call to show before successfully loading an ad.");
     }
 
+    UnityAdsShowOptions unityAdsShowOptions = new UnityAdsShowOptions();
+    unityAdsShowOptions.setObjectId(objectId);
```

Comment:
need to pass unityAdsShowOptions in line 171 :)

Reply:
oops! Good catch! :)